### DISCUSSION TITLE
There is no instructions on how to use this library...

### DISCUSSION THREAD

**Discussion by ronildo** [mathuo/dockview#551] [General]
I'm wondering if I couldn't find it, or if there are no instructions on how to get a project running with dockview.

For example, I'm trying to understand how I can initialize a vanilla project, but I couldn't find anything about it.

![Screenshot 2024-03-13 at 8 35 49 AM](https://github.com/mathuo/dockview/assets/622159/20d867b5-3f61-46f5-acb1-7236d0d322c6)

**Comment by ronildo** [comment]
I found one comment with a link that would have instructions on how to use the vanilla version, but getting a 404.

![Screenshot 2024-03-13 at 8 56 49 AM](https://github.com/mathuo/dockview/assets/622159/284b352f-efc4-4c28-b2e1-9a7dfe084206)


ps: this is where I found the link: https://github.com/mathuo/dockview/discussions/239

**Comment by LeoDroves** [comment]
The best advice that I can give you is take a look at the examples: https://github.com/mathuo/dockview/tree/master/packages/docs/sandboxes
For you case you will probably find something interesting at https://github.com/mathuo/dockview/tree/master/packages/docs/sandboxes/javascript/vanilla-dockview

**Reply by ronildo** [reply]
Thanks @LeoDroves

**Reply by Sedighzade** [reply]
> The best advice that I can give you is take a look at the examples: https://github.com/mathuo/dockview/tree/master/packages/docs/sandboxes For you case you will probably find something interesting at https://github.com/mathuo/dockview/tree/master/packages/docs/sandboxes/javascript/vanilla-dockview

Find Not found!

**Comment by shardworkx-ui** [comment]
I tried the Typescript examples, but opening index.ts in my browser just showed source; running "node index.ts" or "tsc index.ts" just gave errors.  What am I supposed to do?
